### PR TITLE
171648159-story(sequelize): setup sequelize

### PR DESCRIPTION
**Pull request by brianineza01**
#### Description

> Sets up the Sequelize ORM

Please select the relevant option
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Others (cosmetics, styling, improvements)
- [ ] This change requires a documentation update

#### How Has This Been Tested?

- [ ] Unit
- [ ] Integration
- [ ] End-to-end
- [x] No testing required

#### How to Test
**Test using the following commands**
> npm run connectdb
> npm run sequelizetest
 after running those commands check your database and the console 
#### STORY ID
#171648159

[Finishes #171648159]

### CODE REVIEW

**Review by Agnes-N — APPROVED** [review]

**Review by Irutingabo — APPROVED** [review]

**Review by YankyJayChris — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "connectdb": "nodemon --exec babel-node ./server/config/dbconnect.js "
+    "connectdb": "nodemon --exec babel-node ./server/config/dbconnect.js ",
+    "sequelizetest": "sequelize db:migrate & sequelize db:seed:all"
```
Comment (deschantkn) [inline package.json]: I would suggest `migrate` as a better name for this command

**Review by deschantkn — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (brianineza01) [reply package.json]: The changes were added

File: .sequelizerc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const path = require('path')
+
+module.exports = {
+  config: path.resolve('./server/config', 'config.js'),
+  'models-path': path.resolve('./server/models'),
+  'seeders-path': path.resolve('./seeders'),
+  'migrations-path': path.resolve('./migrations'),
```
Comment (deschantkn) [inline .sequelizerc]: I just noticed this folder structure @brianineza01 .  In order to have better organisation in the project, I want to suggest this folder structure. A `sequelize` folder at the root of the project structured as follows:

```
- sequelize
   - models
   - seeders
   - migrations
```

File: server/config/config.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (deschantkn) [inline server/config/config.js]: It would be less confusing if you renamed this fille `environments.js` @brianineza01

**Review by deschantkn — CHANGES_REQUESTED** [review]

File: .sequelizerc (hunk 2 above)
Comment (brianineza01) [reply .sequelizerc]: The folder structure was changed

File: server/config/config.js (hunk 3 above)
Comment (brianineza01) [reply server/config/config.js]: Done that also

**Review by deschantkn — APPROVED** [review]
Great job @brianineza01

**Pull request merged**